code_schemes/kakuma_s01e04_reasons.json

```
@@ -3,6 +3,54 @@
   "Name": "KAKUMA S01E04 Reasons",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-68025c64",
+      "CodeType": "Normal",
+      "DisplayText": "edu - equal responsibilities rights",
```

```suggestion
      "DisplayText": "edu - equal responsibilities/rights",
```
I don't think there's any reason not to use '/' in the display text

---

updated thanks to use or in both cases